client/me/help/help-search/test/index.js

```diff
@@ -18,8 +19,7 @@ import { HelpSearch } from '../';
 jest.mock( '', () => ( {} ) );
```

Comment:
😕 What's going on here @ramonjd?

Let's please drop this if possible.

Reply:
Looks like a mistake. Thanks for picking it up. I'll fix.